template/src/components/utils.js

```diff
@@ -8,3 +8,23 @@ export function debounce(fn, ms) {
     }, ms);
   };
 }
+
```

Comment:
Lets move this to `lib`

Reply:
Nit: Use this https://github.com/niksy/throttle-debounce#readme for both throttle and debounce

Reply:
I considered using libs, but then I thought... why adding a new dependency for something some small?